aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1120,6 +1120,27 @@ class AztecText : EditText, TextWatcher {
         }
     }
 
+    fun applyComment(comment: AztecCommentSpan.Comment) {
+        editableText.replace(selectionStart, selectionEnd, "\n" + comment.html + "\n")
+        setSelection(selectionEnd - 1 - comment.html.length, selectionEnd - 1)
+        removeInlineStylesFromRange(selectionStart, selectionEnd)
```

Comment:
If you add comment right after inline style (Place cursor after **Bold** and add comment) the resulting html will look like this:
```
<b>Bold</b>
<br>
<!--more-->
<br>
<b></b>
```

Maybe we can fix it by extending range of removeInlineStylesFromRange by one? (selectionEnd+1)